### DISCUSSION TITLE
Folia support

### DISCUSSION THREAD

**Discussion by Distolfix** [BG-Software-LLC/SuperiorSkyblock2#1750] [Pending Feature Requests]
add folia support

https://papermc.io/software/folia

**Comment by OmerBenGera** [comment]
Will be added once it is more stable

**Reply by gabrielecabrini** [reply]
If you need some help I can contribute to add Folia support now that's stable

**Reply by HYDeath** [reply]
> Will be added once it is more stable

Currently looking forward to supporting Folia.

**Reply by Sismei** [reply]
need
